apps/judicial-system/backend/src/app/formatters/serviceCertificatePdf.ts

```
@@ -166,13 +179,59 @@ export const createServiceCertificate = (
+      break
+    case DefenderChoice.WAIVE:
+      defenderChoiceText = 'Ég óska ekki eftir verjanda.'
+      break
+    case DefenderChoice.DELEGATE:
+      defenderChoiceText =
+        'Ég fel dómara málsins að tilnefna og skipa mér verjanda.'
+      break
+    case DefenderChoice.DELAY:
+    default:
+      defenderChoiceText =
+        'Ég óska eftir fresti fram að þingfestingu til þess að tilnefna verjanda.'
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)